projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -72,14 +72,27 @@ jQuery( document ).ready( function ( $ ) {
 				html +=
 					'<a href="#" data-module="' +
 					envelope.activate_module +
-					'" type="button" class="jitm-button is-compact is-primary" data-jptracks-name="nudge_click" data-jptracks-prop="jitm-' +
+					'" data-settings_link="' + envelope.module_settings_link +
+					'" type="button" class="jitm-button is-compact is-primary jptracks" data-jptracks-name="nudge_click" data-jptracks-prop="jitm-' +
```

Comment:
We actually don't need the `jptracks` class on buttons with the `jitm-button` class anymore - these buttons are tracked from within this file now.